dataplug/app/com/hubofallthings/dataplug/utils/JwtPhataAuthenticatedAction.scala

```diff
@@ -36,7 +36,7 @@ class JwtPhataAuthenticatedAction @Inject() (
   val parser = bodyParsers.default
 
   def invokeBlock[A](request: Request[A], block: (JwtPhataAuthenticatedRequest[A]) => Future[Result]): Future[Result] = {
-    request.headers.get("X-Auth-Token")
+    request.headers.get("x-auth-token")
```

Comment:
Why is this change necessary?

Reply:
It's not "fixing" anything so it's not necessary at all. I though since we use lowercase, at least in my experience, we should replace them to lowercase as well